### ISSUE TITLE
loss and accuracy not stable (ucf-101, res18)

### ISSUE THREAD

**Issue by emergencyd** [kenshohara/3D-ResNets-PyTorch#125]
![image](https://user-images.githubusercontent.com/16423998/51017344-bfed3f00-15ae-11e9-8d80-e814fbe8c042.png)

all the results are from epoch 200.
Is that normal or there is something wrong?

**Comment by kenshohara** [comment]
If the results are by training from scratch, they seem to be normal.

**Comment by hirokatsukataoka16** [comment]
As we mentioned in the paper, you already have a better option. You should employ a Kinetics pre-trained model rather than from scratch.
Our paper: https://arxiv.org/pdf/1711.09577.pdf
Pre-trained models: https://drive.google.com/drive/folders/1zvl89AgFAApbH0At-gMuZSeQB_LpNP-M
